src/core/Akka.Tests.Performance/Dispatch/MailboxMemoryFootprintSpec.cs

```diff
@@ -17,24 +20,43 @@ public class MailboxMemoryFootprintSpec
 
         private const string CreateThroughputCounter = "MailboxCreateThroughput";
         private const int MailboxCreateNumber = 10000;
+        private ActorSystem _actorSystem = ActorSystem.Create("Testing");
```

Comment:
Had to adjust the mailbox memory footprint specs to support the new `MailboxType` class.